### ISSUE TITLE
JUnit report- testsuite name

### ISSUE THREAD

**Issue by IrekW** [galenframework/galen#479]
Hi,
I'm using the junit report on jenkins pipeline together with some other reports. All junit reports are then collected on one summarised report. The Galen xml report file starts with <testsuites> tag while other reports have also some names and other attributes (for example <testsuite name="PhantomJS 2.1.1 (Windows 8 0.0.0)" tests="23" errors="0" failures="0" time="68.914">). As a result, the Galen report is named "root" (on the level of testsuites) where other reports have their own names. Of course "Galen Suite" is inside this "root", one level lower.
Is it possible to specify the name also for the root testsuites in XML so jenkins will recognize it as Galen and not root? I know I can manually modify "galen.jar/junit-report/junit-report.ftl.xml" but I want to avoid it.

**Comment by hypery2k** [comment]
the name Galen Suite is currently hard-coded for [JUnit](https://github.com/galenframework/galen/blob/master/galen-core/src/main/resources/junit-report/junit-report.ftl.xml#L3) and [TestNG](https://github.com/galenframework/galen/blob/master/galen-core/src/main/resources/testng-report/testng-report.ftl.xml#L3).

You want to add the browser info there, right?

**Comment by IrekW** [comment]
No, I just wanted to place it on the same level as other reports so to see that this is Galen report and not some unknown "root". Galen reports have one level more because of testsuites tag so it is added by Jenkins as root/Galen Suite. Other reports just start without this root, so one click closer.
